services/client/src/styles/_variables.scss

```
@@ -0,0 +1,38 @@
+// ── Colors ──────────────────────────────────────────────────
+
+$color-primary: #0b6394;
+$color-primary-dark: #084b6f;
+$color-success: #2e7d32;
+$color-error: #b91c1c;
+$color-warn: #b45309;
+// for text
```

Stylelint (17.14.0)

[error] Expected empty line before comment (scss/double-slash-comment-empty-line-before)

(scss/double-slash-comment-empty-line-before)